<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1021</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>540</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>表格数据</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_sort">
          <property name="title">
           <string>排序</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>排序字段</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_sort"/>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_rise">
             <property name="text">
              <string>升序</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_down">
             <property name="text">
              <string>降序</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_filter">
          <property name="title">
           <string>数据过滤</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton__man">
             <property name="text">
              <string>男</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_womon">
             <property name="text">
              <string>女</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_showAll">
             <property name="text">
              <string>全显示</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>当前记录</string>
     </property>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>461</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>备注</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>231</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>照片</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>工  号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>姓  名</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>性  别</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_sex">
         <item>
          <property name="text">
           <string>男</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>女</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>出生日期</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_birthDay">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>2000-12-3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2000-3-26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2000-6-1</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>出生省份</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_birthProvince">
         <item>
          <property name="text">
           <string>江苏省</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>北京市</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>广东省</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>部  门</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_department">
         <item>
          <property name="text">
           <string>程序员</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>运维工程师</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>后端程序员</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>工  资</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="spinBox_salary">
         <property name="maximum">
          <number>9999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_add"/>
   <addaction name="action_insert"/>
   <addaction name="action_delete"/>
   <addaction name="separator"/>
   <addaction name="action_setPhoto"/>
   <addaction name="action_clearPhoto"/>
   <addaction name="action_increase"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="action_cancel"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_add">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_insert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_setPhoto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>设置照片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_clearPhoto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>清除照片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_increase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>涨工资</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_cancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
